tests/unit/test_operator_bonded_tracker.py

```diff
@@ -20,9 +20,6 @@ def test_operator_never_bonded(mocker, get_random_checksum_address):
 
     tracker = OperatorBondedTracker(ursula=ursula)
     try:
-        d = threads.deferToThread(tracker.start)
```

Comment:
How come we no longer start the tracker anymore?

Reply:
it's already running at the time this test is executed so this call had no impact. Also, this test awaits the deferred from `.run` directly in order to make it's assertions, which pass.

Reply:
> t's already running at the time this test is executed

Why is it already running? Is something else calling `start()`?

> Also, this test awaits the deferred from .run directly in order to make it's assertions, which pass.

It does but one additional part of the test is also ensuring that stopping the tracker works (L32)

Reply:
> Why is it already running? Is something else calling start()?

Yes, but I'm not sure what (too out-of-scope to investigate) In general there some background noise in the form of uncontrolled async tasks while running the integration and acceptance suites.  Cleaning them up will speed up test performance and accuracy but i snot causing any issues I can see right now.

> It does but one additional part of the test is also ensuring that stopping the tracker works (L32)

I don't think this test is useful since stop is not really used for normal node operation and no assertions are made about the call to stop.  I kind of see this line as test tear-down not related to "test_operator_never_bonded".

In any case I restored the diff to it's original state.

Reply:
✔️